inc/ui/namespace.php

```diff
@@ -71,22 +82,36 @@ function load_page() {
 	wp_enqueue_script( __NAMESPACE__, plugins_url( '/assets/script.js', Documentation\DIRECTORY . '/wp-is-dumb' ), [ 'highlightjs' ], '2019-04-19' );
 
 	// Determine the current page title.
-	$page = Documentation\get_page_by_id( get_current_group_id(), get_current_page_id() );
+	$page = Documentation\get_page_by_id( get_current_group_id(), get_current_page_id(), get_current_set_id() );
 	if ( $page ) {
 		$GLOBALS['title'] = $page->get_meta( 'title' );
 	} else {
 		$GLOBALS['title'] = __( 'Page Not Found', 'altis' );
 	}
 }
 
+/**
+ * Get the current Set ID.
+ *
+ * @return string Doc set ID if set, otherwise the default set.
+ */
+function get_current_set_id() : string {
+	// @codingStandardsIgnoreLine
+	return $_GET['set'] ?? apply_filters( 'altis.documentation.default.set', 'dev-docs' );
```

Comment:
I'll add the doc block.

This couldend up putting some other docs on the default documentation page if the other docs  `'altis.documentation.sets'` filter was called earlier than Altis' own.

Reply:
I think that's fine. It's not super likely to happen as that filter is added really early but if someone did so it would likely be intentional. We also want to allow for not having the dev docs present e.g. on cloud environments.

I think this would require having a function that returns all doc sets and then the `get_documentation_set()` function could use that to get the list and return a specific one.

The docblock has the coding standards ignore line between it and the `apply_filters()` call so I'm not sure if the WP specific parser or John Blackbourns VSCode addon would pick it up properly. You probably noticed the pattern where `apply_filters()` is used and assigned to a variable, even if it's immediately returned, or used in a conditional. It's a more cautious approach to code markup to try and make life easier for parsers.